django/contrib/postgres/indexes.py

```diff
@@ -43,5 +43,52 @@ def get_sql_create_template_values(self, model, schema_editor, using):
 class GinIndex(Index):
     suffix = 'gin'
 
+    GIN_PENDING_LIST_LOWER = 2**6
```

Comment:
How about removing the `GIN_` prefix? It's already defined on the `GinIndex` class.